bears/coffee_script/CoffeeLintBear.py

```diff
@@ -28,7 +29,7 @@ class CoffeeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', '-f', config_file, filename
+        return '--reporter=raw', '--stdin', '-f', config_file, filename
```

Comment:
ah no :)

2016-06-29 22:58 GMT+01:00 Mischa Krüger <notifications@github.com>:

> In bears/coffee_script/CoffeeLintBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/555#discussion_r69036313>
> :
>
> > @@ -28,7 +29,7 @@ class CoffeeLintBear:
> >
> >      @staticmethod
> >      def create_arguments(filename, file, config_file):
> > -        return '--reporter=raw', '-f', config_file, filename
> > +        return '--reporter=raw', '--stdin', '-f', config_file, filename
>
> as you pass the file via stdin, do you actuall need the filename?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/555/files/1d0c883458331e13c52afcf361aef4f4d4df362f..1fb7b3ef239253b2c5a739749725c8b36fb25602#r69036313>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xonAsXkm1-p6jeZnbXTJtooZRdQX0ks5qQuqjgaJpZM4I9f6t>
> .
>